src/components/tile/Tile.js

```diff
@@ -0,0 +1,9 @@
+function Tile({image, tileTitle, tileText}) {
+    return (<>
+        <img src={image} alt=""/>
+        <h2>{tileTitle}</h2>
+        <p>{tileText}</p>
```

Comment:
Hier zou je {children} kunnen gebruiken om de meerdere tekst vlakken te implementeren. Of zelfs met een </br> en extra <p> veld met property voor dit specifieke geval.